@here/harp-mapview/lib/PickHandler.ts

```diff
@@ -210,11 +210,17 @@ export class PickHandler {
             planeIntersectPosition.add(this.mapView.worldCenter);
 
             const cameraPos = this.mapView.worldCenter.clone().add(this.mapView.camera.position);
-
-            this.mapView.forEachVisibleTile(tile => {
+            const worldExtents = this.mapView.projection.worldExtent(0, 0).max.x;
+            this.mapView.forEachVisibleTile(tileOrProxy => {
+                // This removes any offset from the camera position, such that the intersection
+                // query takes place as if the camera is over the tile at offset 0
```

Comment:
a colon at the end of a sentence is always nice in a comment, especially if it starts in upper case, like a proper sentence.

Reply:
You mean "full stop"? Agreed